sixtyfps_runtime/corelib/items.rs

```diff
@@ -391,6 +393,7 @@ impl Item for TouchArea {
             Self::FIELD_OFFSETS.mouse_y.apply_pin(self).set(pos.y);
         }
         Self::FIELD_OFFSETS.has_hover.apply_pin(self).set(!matches!(event, MouseEvent::MouseExit));
+        Self::FIELD_OFFSETS.cursor_style.apply_pin(self).set(CursorStyle::grab);
```

Comment:
Here you shouldn't set the cursor_style property.
Instead, you should read the property with get(),  and pass that property to `_window`
You want to add interfaces to the `sixtyfps_corelib::window::PlatformWindow` like a set_cursor function there.
then re-implement that in the Qt and GL backend.